rpcs3/Emu/NP/rpcn_config.cpp

```diff
@@ -0,0 +1,77 @@
+﻿#include "stdafx.h"
+#include "rpcn_config.h"
+#include "Emu/System.h"
+
+cfg_rpcn g_cfg_rpcn;
+
+void cfg_rpcn::load()
+{
+	if (fs::file cfg_file{cfg_rpcn::get_path(), fs::read})
+	{
+		from_string(cfg_file.to_string());
+	}
+}
+
+void cfg_rpcn::save() const
+{
+	fs::file(cfg_rpcn::get_path(), fs::rewrite).write(to_string());
+}
+
+std::string cfg_rpcn::get_path()
+{
+	return fs::get_config_dir() + "rpcn.yml";
```

Comment:
I think we should put any new config files into the config folder on windows.
```suggestion
#ifdef _WIN32
	return fs::get_config_dir() + "config/rpcn.yml";
#else
	return fs::get_config_dir() + "rpcn.yml";
#endif
```